sycl/include/CL/sycl/detail/pi.hpp

```diff
@@ -155,6 +155,29 @@ template <> inline void print<>(PiPlatform val) {
   std::cout << "pi_platform : " << val << std::endl;
 }
 
+template <> inline void print<>(PiEvent val) {
+  std::cout << "pi_event : " << val << std::endl;
+}
+
+template <> inline void print<>(PiMem val) {
+  std::cout << "pi_mem : " << val << std::endl;
+}
+
+template <> inline void print<>(PiEvent *val) {
+  std::cout << "pi_event * : " << val;
+  if (val) {
```

Comment:
Why don't we print nullptr if event is nullptr?

Reply:
Fixed, thanks.